mne/simulation/tests/test_source.py

```diff
@@ -146,6 +146,9 @@ def test_simulate_sparse_stc():
                   subjects_dir=subjects_dir)  # no subject
     pytest.raises(ValueError, simulate_sparse_stc, fwd['src'], len(labels),
                   times, labels=labels, location='foo')  # bad location
+    pytest.raises(ValueError, simulate_sparse_stc, fwd['src'], len(labels) + 1,
```

Comment:
The modern/clearer way to do this is:
```
with pytest.raises(ValueError, match='some regex str'):
    simulate_sparse_stc(...)
```
It's clearer and more precise.

Also there is a PEP8 error

https://travis-ci.org/mne-tools/mne-python/jobs/496513546#L3207

Otherwise LGTM